### PR TITLE
Skip make public for uniform storage permissions

### PR DESCRIPTION
**Pull request by jheth**
- Bump google-cloud/storage

### CODE REVIEW

**Review by knownasilya — APPROVED** [review]

**Comment by knownasilya** [comment]
Thanks for this 🙂 looks like we'll have to drop Node < 10, for the Google dependency. About time anyway. Think you could make that change?

File: .travis.yml [hunk 1]
```diff
@@ -37,7 +37,7 @@ matrix:
 
 before_install:
   - npm config set spin false
-  - npm install -g npm@6
+  - npm install -g npm@10
```
Comment (knownasilya) [inline .travis.yml]: Looks like this was a mistake, there is no npm v10, I think it's stilll at v6.

**Comment by jheth** [comment]
Finally got it! Thanks for the help.

**Comment by knownasilya** [comment]
Thanks!

**Comment by jheth** [comment]
@knownasilya  could you cut a tag for this new version? Should be v0.5.0 since it's not backward compat with older node versions.

**Comment by knownasilya** [comment]
Will try to release tonight. Will probably make 1.0 since it's pretty stable.

**Comment by jheth** [comment]
@knownasilya can you add the 1.0 tag? I'm wanting to point back to your repo instead of using our fork.

**Comment by knownasilya** [comment]
Thanks for the reminder

**Comment by knownasilya** [comment]
Released

**Comment by knownasilya** [comment]
Actually use 1.0.1, fixes a bug where `makePublic` wasn't passed down as a config.

**Comment by jheth** [comment]
Thanks so much!

**Pull request merged**